packages/tools/client-debugger/client-debugger/src/Registry.ts

```diff
@@ -14,35 +16,103 @@ import { IFluidClientDebugger } from "./IFluidClientDebugger";
  * @public
  */
 export interface FluidClientDebuggerProps {
-    /**
-     * The Container with which the debugger will be associated.
-     */
-    container: IContainer;
-
-    /**
-     * The ID of {@link FluidClientDebuggerProps.container | the Container}.
-     */
-    containerId: string;
-
-    /**
-     * Optional: Data belonging to {@link FluidClientDebuggerProps.container | the Container}.
-     *
-     * @remarks The debugger will not mutate this data.
-     */
-    containerData?: IFluidLoadable | Record<string, IFluidLoadable>;
-
-    /**
-     * Optional: Nickname for {@link FluidClientDebuggerProps.container | the Container} / debugger instance.
-     *
-     * @remarks
-     *
-     * Associated tooling may take advantage of this to differentiate between debugger instances using
-     * semantically meaningful information.
-     *
-     * If not provided, the {@link FluidClientDebuggerProps.containerId} will be used for the purpose of distinguising
-     * debugger instances.
-     */
-    containerNickname?: string;
+	/**
```

Comment:
Nit: Looks like there are some whitespace changes here. Be sure to run `npm run format` inside the package - should clean up whitespace diffs.

Reply:
run the format. Thanks for catching this up.